spec/services/github_handler_spec.rb

```diff
@@ -48,23 +49,35 @@
       end
 
       it 'does not duplicate same user with different login' do
-        user
         expect {
-          github_handler.create_or_find_user(payload['requested_reviewer'])
+          github_handler.create_or_find_user(payload.requested_reviewer)
         }.to change(User, :count).by(0)
       end
     end
 
     describe 'pull request not existing' do
       it 'adds pr to the DB' do
         expect {
-          github_handler.create_or_find_pr(payload['pull_request'])
+          github_handler.opened
```

Comment:
I'd reorganize the test file like

`describe '#opened' do`
`describe '#closed' do`
`describe '#review_requested' do`
`describe '#review_removal' do`

http://www.betterspecs.org/#describe